src/main.rs

```diff
@@ -510,12 +494,12 @@ fn run() -> Result<(), failure::Error> {
         match kv_matches.subcommand() {
             ("create", Some(create_matches)) => {
                 let env = create_matches.value_of("env");
-                let target = manifest.get_target(env, false)?;
+                let target = manifest.get_target(env)?;
```

Comment:
nit: fwiw I prefer this approach (`let env ...; let target = manifest.get_target(env)`) to the one-liner